infra/template.html

```diff
@@ -30,7 +30,8 @@ <h1 class="title">$title$</h1>
 <p class="author">$author$</p>
 $endfor$
 <a href="https://twitter.com/browserbook">Twitter</a> · <a href="/blog/">Blog</a> ·
-<a href="https://patreon.com/browserengineering">Patreon</a>
+<a href="https://patreon.com/browserengineering">Patreon</a> -
```

Comment:
Uhh, can we do the center dot here instead of the dash?

Reply:
Fixed